install-poetry.py

```diff
@@ -134,38 +143,29 @@ def string_to_bool(value):
     return value in {"true", "1", "y", "yes"}
 
 
-def data_dir(version: Optional[str] = None) -> Path:
+def data_dir() -> Path:
```

Comment:
Any reason for not pulling this and `bin_dir` functions into the `Installer` class directly? They are only used there anyway.

Reply:
This is a stepping stone to a `System` class that will encapsulate these values and some others. I am not sure if this will eventually be a series of PRs, or one big one with reviewable commits -- but that evolution will make itself more apparent over time.